api-reference/webhook/get-started.mdx

```diff
@@ -0,0 +1,154 @@
+
+### Testing your Endpoint
+
+The easiest way to be more confident in your endpoint configuration is to start receiving events as quickly as possible.
+
+That's why we have a "Testing" tab for you to send example events to your endpoint.
+
+![testing endpoint](/images/webhook/testing-endpoint.png)
+
+After sending an example event, you can click into the message to view the message payload, all of the message attempts, and whether it succeeded or failed.
+
+## Retries
+Svix attempts to deliver each webhook message based on a retry schedule with exponential backoff.
+
+### The schedule
+Each message is attempted based on the following schedule, where each period is started following the failure of the preceding attempt:
+
+-   Immediately
+-   5 seconds
+-   5 minutes
+-   30 minutes
+-   2 hours
+-   5 hours
+-   10 hours
+-   10 hours (in addition to the previous)
+
+If an endpoint is removed or disabled delivery attempts to the endpoint will be disabled as well.
+
+For example, an attempt that fails three times before eventually succeeding will be delivered roughly 35 minutes and 5 seconds following the first attempt.
+
+### Disabling failing endpoints
+If all attempts to a specific endpoint fail for a period of 5 days, the endpoint will be disabled and a webhook (`EndpointDisabledEvent`) will be sent to your account (not to the failing endpoint).
+
+### Manual retries
+You can also use the application portal to manually retry each message at any time, or automatically retry ("Recover") all failed messages starting from a given date.
+
+## Troubleshooting Tips
+There are some common reasons why your webhook endpoint is failing:
+
+### Not using the raw payload body
+This is the most common issue. When generating the signed content, we use the raw string body of the message payload.
+
+If you convert JSON payloads into strings using methods like stringify, different implementations may produce different string representations of the JSON object, which can lead to discrepancies when verifying the signature. It's crucial to verify the payload exactly as it was sent, byte-for-byte or string-for-string, to ensure accurate verification.
+
+### Missing the secret key
+From time to time we see people simple using the wrong secret key. Remember that keys are unique to endpoints.
+
+### Sending the wrong response codes
+When we receive a response with a 2xx status code, we interpret that as a successful delivery even if you indicate a failure in the response payload. Make sure to use the right response status codes so we know when message are supposed to succeed vs fail.
+
+### Responses timing out
+We will consider any message that fails to send a response within 15s a failed message. If your endpoint is also processing complicated workflows, it may timeout and result in failed messages.
```

Comment:
```suggestion
We will consider any message that fails to send a response within 15s to be a failed message. If your endpoint is also processing complicated workflows, it may timeout and result in failed messages.
```